pkg/options/options.go

```diff
@@ -39,6 +40,8 @@ func NewNetdConfig() *NetdConfig {
 func (nc *NetdConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&nc.EnablePolicyRouting, "enable-policy-routing", false,
 		"Enable policy routing.")
+	fs.BoolVar(&nc.EnableSourceValidMark, "enable-source-valid-mark", true,
```

Comment:
@jingyuanliang Per our offline discussion - updated the default to `true` to match up the previous behavior.